options-gen/cmd.go

```diff
@@ -120,3 +124,26 @@ func Run(opts Options) error {
 
 	return nil
 }
+
+func applyExcludes(specs *generator.GetOptionSpecRes, excludes []string) error {
+	for _, pattern := range excludes {
+		reg, err := regexp.Compile(pattern)
```

Comment:
You should compile all patterns first (before any package parse started). This is like a request validation procedure. This is allows us to make a fail-fast solution and exit from program as fast as we can.